docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/header/header.stories.js

```diff
@@ -62,6 +63,10 @@ export const Header = (knobTab) => {
       primaryNavigationKnobTab,
     );
     generalKnobs.primary_navigation_dropdown_columns_fill = boolean('Fill width for missing columns', false, primaryNavigationKnobTab);
+    generalKnobs.search = boolean('Show Search', true, SearchLinkKnobTab) ? {
```

Comment:
could you please rename `search` to `with_search` and title of the knob to `With Search`. This is to distiguish that `search` (which is a component name) is different from the boolean field to show or hide it.